### PR TITLE
Update tabs navigation layout to new design spec

### LINKED ISSUE #3086: [Enhancement]Tabs navigation layout
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
"Overfør" should be a navigation entry in Tabs. "Konto" should be removed.

Selected tabs is today highlighted (selected / deselected) with a "filled" icon. This could be an issue with "Overfør" because the icon is hard to make "filled"

We should consider how we could show the two states (selected / deselected). 

- Change icons for the states (black / grey)
- Change opacity on deselcted icons

Change font size from 10pt. regular to **12pt. medium**

**LAYOUT**

Tabbar, Overblik

https://zpl.io/1yjxZWB

**New icon**
"menu-no-decoration"
menu-no-decoration

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [x] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [x] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [x] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [x] Signal to others you are working on the issue by assigning yourself.
- [x] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [x] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [x] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [x] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

### PR DESCRIPTION
**Pull request by jakobe**
## Which issue does this PR close?

This PR closes #3086 

## What is the new behavior?

- Align with [updated design spec](https://zpl.io/1yjxZWB): dimmed icon and text color on unselected tabs
- Add new `menu-no-decoration` icon
- Refactor spacing on tabs
- Update docs with new layout and icon

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Are there any additional context?

No.

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [x] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [x] Request that the changes are code-reviewed 
- [x] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: apps/cookbook/src/app/examples/page-example/fixed-footer-tabs/fixed-footer-tabs-example.component.ts [hunk 1]
```diff
@@ -8,27 +8,24 @@ const template = `<!-- This container is only needed due to layout of larger scr
   </div>
 
   <kirby-tab-bar>
-    <kirby-tab-button routerLink="dashboard">
-      <kirby-icon name="person-outline"></kirby-icon>
-      <kirby-icon name="person" selected-tab></kirby-icon>
-      Dashboard
+    <kirby-tab-button routerLink="overview">
+      <kirby-icon name="kirby"></kirby-icon>
```
Comment (mark-drastrup) [inline apps/cookbook/src/app/examples/page-example/fixed-footer-tabs/fixed-footer-tabs-example.component.ts]: I just went to the documentation for icon to see what icon `kirby` is. I noticed that it's the exact same icon as `overview-outline`. I propose that `overview-outline` is used instead, as that name is more descriptive, and that we create an issue on removing the icon named `kirby`, so we don't have any duplicates :)

File: apps/cookbook/src/app/examples/tabs-example/tabs-example.component.html [hunk 2]
```diff
@@ -6,27 +6,24 @@
   </div>
 
   <kirby-tab-bar>
-    <kirby-tab-button routerLink="dashboard">
-      <kirby-icon name="person-outline"></kirby-icon>
-      <kirby-icon name="person" selected-tab></kirby-icon>
-      Dashboard
+    <kirby-tab-button routerLink="overview">
+      <kirby-icon name="kirby"></kirby-icon>
```
Comment (mark-drastrup) [inline apps/cookbook/src/app/examples/tabs-example/tabs-example.component.html]: Same as above :)

File: apps/cookbook/src/app/examples/page-example/fixed-footer-tabs/fixed-footer-tabs-example.component.ts (hunk 1 above)
Comment (jakobe) [reply apps/cookbook/src/app/examples/page-example/fixed-footer-tabs/fixed-footer-tabs-example.component.ts]: Good catch @mark-drastrup ! 👌🏻 🔍 
I've updated the docs example to `overview-outline`: https://github.com/kirbydesign/designsystem/pull/3126/commits/e8ee13e34ac6a5e6cac4af4eb701c8971d218074

Regarding your 2nd comment: We shouldn't do that 😅 
We use the Kirby icon everywhere for examples - `overview` and `overview-outline` currently is the same icon, but that could potentially change later...

File: libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts [hunk 3]
```diff
@@ -1,24 +1,184 @@
...
+      expect(ionTabButton).toHaveComputedStyle({
+        color: getTextColor('semi-dark'),
+      });
+    });
+
+    it('should set correct color on selected tab', () => {
+      ionTabButton.selected = true;
+
+      expect(ionTabButton).toHaveComputedStyle({
+        color: DesignTokenHelper.getTextColor('semi-dark'),
```
Comment (mark-drastrup) [inline libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts]:
```suggestion
        color: getTextColor('semi-dark'),
```

File: libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts [hunk 4]
```diff
@@ -1,24 +1,184 @@
...
+      });
+    });
+
+    it('should set correct color on selected tab', () => {
+      ionTabButton.selected = true;
+
+      expect(ionTabButton).toHaveComputedStyle({
+        color: DesignTokenHelper.getTextColor('semi-dark'),
```
Comment (mark-drastrup) [inline libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts]:
```suggestion
        color: getTextColor('semi-dark'),
```

File: libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts [hunk 5]
```diff
@@ -1,24 +1,184 @@
...
+      });
+    });
+
+    it('should set correct color on selected tab', () => {
```
Comment (mark-drastrup) [inline libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts]: Isn't this a duplicate of the previous test on line 43?

File: libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts (hunk 3 above)
Comment (mark-drastrup) [reply libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts]: Just double checking - should `color` be `semi-dark` both when the tab is unselected and selected?

**Review by mark-drastrup — CHANGES_REQUESTED** [review]
Looks really great! I just have a couple of more nitty gritty comments :)

File: apps/cookbook/src/app/examples/page-example/fixed-footer-tabs/fixed-footer-tabs-example.component.ts (hunk 1 above)
Comment (mark-drastrup) [reply apps/cookbook/src/app/examples/page-example/fixed-footer-tabs/fixed-footer-tabs-example.component.ts]: @jakobe Ahh I see 😅 Yeah let's keep it the way it is then 😄

File: libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts (hunk 4 above)
Comment (jakobe) [reply libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts]: Good catch! 👌🏻 🔍 
Fixed 👍🏻

File: libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts (hunk 3 above)
Comment (jakobe) [reply libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts]: > Just double checking - should `color` be `semi-dark` both when the tab is unselected and selected?

Nope, that's a copy-paste mistake 🙈  - good catch! Fixed 👍🏻

File: libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts (hunk 5 above)
Comment (jakobe) [reply libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts]: Yes it is 🙈  Good catch! 👏🏻 
Removed 🔥 👍🏻

**Review by mark-drastrup — DISMISSED** [review]
Approved! ✅✅✅

**Review by mark-drastrup — DISMISSED** [review]

**Review by mark-drastrup — APPROVED** [review]

**Pull request merged**